index.html

```diff
@@ -0,0 +1,226 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Сервис для поиска попутчиков для путешествий - Погнали</title>
+</head>
+<body>
+  <header class="page-header">
+    <a class="page-header__logo">
+      <img class="page-header__logo-image" src="https://via.placeholder.com/100x19.png" alt="Логотип Погнали" width="100" height="19">
+    </a>
+    <nav class="main-nav site-list">
+      <button class="main-nav__toggle">Открыть меню</button>
+      <div class="main-nav__wrapper">
+        <ul class="main-nav__list site-list">
+          <li class="site-list__item">
+            <a>Главная</a>
+          </li>
+        </ul>
+      </div>
+      <ul class="main-nav user-list">
+        <li class="user-list__item">
+          <a href="tel:+74951234567"></a>
+        </li>
+        <li class="user-list__item">
+          <a href="mailto:info@site.com"></a>
+        </li>
+        <li class="user-list__item">
+          <a href="sign-in.html">Авторизация</a></li>
+      </ul>
+    </nav>
+  </header>
+  <main class="page-main">
+    <h1 class="visually-hidden">Сервис для поиска попутчиков для путешествий - Погнали</h1>
+    <header class="main-header">
+      <h2 class="main-header__title">В путешествие с крутыми попутчиками!</h2>
+      <div class="main-header__review">
+        <blockquote>
+          <p>«Лучшее приложение, что я видел в жизни!»</p>
+          <b>Мурат Эльтазинглавный редактор газеты «гаджетлайф»</b>
+        </blockquote>
+      </div>
+      <div class="main-header__step">
+        <ul class="step-list">
+          <li class="step-item"><a href="#">Выберите направление</a></li>
+          <li class="step-item"><a href="#">Изучите идеи путешественников</a></li>
+          <li class="step-item"><a href="#">Находите тех, кто похож на вас</a></li>
+          <li class="step-item"><a href="#">Путешествуйте вместе!</a></li>
+        </ul>
+      </div>
+    </header>
+    <section class="about-servise">
+      <h2 class="about-servise__title">О сервисе</h2>
+      <p class="about-servise__text">
+        Любое путешествие веселее
+        с попутчиками, которые на одной волне с тобой! Делитесь своими идеями путешествий и черпайте вдохновение в чужих.
+      </p>
+      <p class="about-servise__text--bold">
+        А если найдете кого-то близкого по духу — скорее зовите в совместный трип!
+      </p>
+    </section>
+    <section class="directions">
+      <h2 class="directions__title">Направления</h2>
+      <p class="directions__text">Мы не продаем туры и ничего не рекламируем. Люди сами пишут о странах,
+        куда хотели бы отправиться и чем можно там заняться.</p>
+      <div class="directions-point">
+        <ul class="directions-point__list">
+          <li class="directions-point__item">
```

Comment:
```suggestion
          <li class="directions-point__item point">
```
И снизу уже наследоваться от класса `point`.

Это рекомендация, так ты аналогично получишь еллемент, который можешь использовать отдельно.